main/consts.js

```diff
@@ -1,10 +1,31 @@
 const os = require('os')
 const packageJson = require('../package.json')
+const path = require('path')
+const { app } = require('electron')
 
 module.exports = Object.freeze({
+  CACHE_HOME: getCacheHome(),
   IS_MAC: os.platform() === 'darwin',
   IS_WIN: os.platform() === 'win32',
   IS_APPIMAGE: typeof process.env.APPIMAGE !== 'undefined',
   STATION_VERSION: packageJson.version,
   ELECTRON_VERSION: process.versions.electron
 })
+
+// Replace with `app.get('localUserData')` after this PR is landed & released:
+// https://github.com/electron/electron/pull/34337
+function getCacheHome () {
+  const platform = os.platform()
+  switch (platform) {
+    case 'darwin': // macOS
+      return path.join(app.getPath('home'), 'Library', 'Caches', app.name)
+    case 'win32':
+      if (!process.env.LOCALAPPDATA) throw new Error('Unsupported Windows environment: LOCALAPPDATA must be set.')
+      return path.join(process.env.LOCALAPPDATA, app.name)
+    case 'linux':
+      if (!process.env.XDG_CACHE_HOME) throw new Error('Unsupported Linux environment: XDG_CACHE_HOME must be set.')
+      return path.join(process.env.XDG_CACHE_HOME, app.name)
```

Comment:
A discussion point: there seems to be several different naming conventions used for cache files on macOS.

- Some application use `appId` - a reverse hostname, e.g. `com.apple.appstore` and `com.skype.skype`.
- Other apps use the application/product name, e.g. `Homebrew`, `Marked 2`, `Microsoft Edge`
- I found also directories called after the vendor, e.g. `Microsoft` or `Mozilla`
- Last but not least, some tools use just the package name, e.g. `esbuild`, `esbuild`, `electron`, etc.

Apparently, there isn't a rule that everybody would consistently follow.

In my proposal here, I am using the user-friendly app name `Filecoin Station`.

```
Using Saturn L2 Node binary: /Users/bajtos/src/saturn/filecoin-station/build/saturn/l2node-darwin-x64/saturn-L2-node
Starting Saturn node...
[SATURN] Using root dir /Users/bajtos/Library/Caches/Filecoin Station/saturn
```

It seems some other parts of our stack are already using the `appId`, I found `~/Library/Caches/io.filecoin.station` in my home dir. As a result, Station users will get two new directories created in their cache home. I feel this is a minor detail we can ignore until somebody complains.

Reply:
I agree with waiting until someone complains, and I don't see any big advantages or disadvantages between the naming patterns.